functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/BulkJobActionsSpec.groovy

```diff
@@ -0,0 +1,215 @@
+
+        JobListPage jobsListPage = page JobListPage
+        jobsListPage.loadPathToNextUI SELENIUM_BASIC_PROJECT
+        jobsListPage.go()
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.jobsActionsButton).click()
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkEditButton).click()
+
+        when:
+        jobsListPage.waitForNumberOfElementsToBeMoreThan(jobsListPage.bulkJobRowItemsBy, 1)
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkSelectAllButton).click()
+
+        then:
+        def bulkPerformActionButton = jobsListPage.getBulkPerformActionButtonOnceClickable()
+        jobsListPage.waitIgnoringForElementToBeClickable(bulkPerformActionButton).click()
+
+        when:
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkEnableSchedulesAction).click()
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkConfirmActionYesButton).click()
+
+        then:
+        jobRespondsTrueTo(jobId1.toString()) { it.scheduleEnabled }
+    }
+
+    def "bulk disable schedule action disables the schedule"() {
+        given:
+        jobId1 = generateJob(["schedule-enabled": "true", "schedule-crontab": CRON_SCHEDULE_THAT_WONT_EXECUTE], "api-test-executions-running-scheduled.xml")
+        jobId2 = generateJob(["schedule-enabled": "true", "schedule-crontab": CRON_SCHEDULE_THAT_WONT_EXECUTE], "api-test-executions-running-scheduled.xml")
+
+        JobListPage jobsListPage = page JobListPage
+        jobsListPage.loadPathToNextUI SELENIUM_BASIC_PROJECT
+        jobsListPage.go()
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.jobsActionsButton).click()
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkEditButton).click()
+
+        when:
+        jobsListPage.waitForNumberOfElementsToBeMoreThan(jobsListPage.bulkJobRowItemsBy, 1)
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkSelectAllButton).click()
+
+        then:
+        def bulkPerformActionButton = jobsListPage.getBulkPerformActionButtonOnceClickable()
+        jobsListPage.waitIgnoringForElementToBeClickable(bulkPerformActionButton).click()
+
+        when:
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkDisableSchedulesAction).click()
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkConfirmActionYesButton).click()
+
+        then:
+        jobRespondsTrueTo(jobId1.toString()) { !it.scheduleEnabled }
+    }
+
+    def "job can be executed from the Job List Page modal"() {
+        given:
+        String jobToExecute = generateJob(["opt1-required": "false", "opt2-required": "false"])
+
+        JobListPage jobsListPage = page JobListPage
+
+        // TODO: Determine why the run button fails to open a modal in NextUI
+        //        jobsListPage.loadPathToNextUI SELENIUM_BASIC_PROJECT
```

Comment:
Not sure if this comment refers to a bug in nextUI, or a problem with selenium and nextUI modal. But in either way its very probable this TODO will be forgotten forever. Maybe create a jira ticket for either a bug or flaky test would be a better option.

Reply:
Exactly! It has already been forgotten by me... even before getting merged. :-)
I was supposed to figure it out before starting the review, but I forgot!
Thank you for noticing this.